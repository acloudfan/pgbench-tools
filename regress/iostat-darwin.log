2020-08-30 21:02:32.283081	              disk0       cpu
2020-08-30 21:02:32.283180	    KB/t  tps  MB/s  us sy id
2020-08-30 21:02:32.283212	   26.49   18  0.47   2  1 96
2020-08-30 21:02:33.283271	   28.40   20  0.55  12  4 84
2020-08-30 21:02:34.286027	   13.60   10  0.13  11  4 85
2020-08-30 21:02:35.286896	   30.96   23  0.69   9  4 87
2020-08-30 21:02:36.290639	   90.00    4  0.35   6  4 90
2020-08-30 21:02:37.290742	   23.37   63  1.44   7  4 89
2020-08-30 21:02:38.291489	  166.61   46  7.48   7  4 89
2020-08-30 21:02:39.291605	    0.00    0  0.00   7  4 89
2020-08-30 21:02:40.291813	   24.00    3  0.07   5  4 91
2020-08-30 21:02:41.291947	    4.83  247  1.16   6  4 90
2020-08-30 21:02:42.292114	   62.67   12  0.73   7  4 88
2020-08-30 21:02:43.292373	    0.00    0  0.00   5  4 91
2020-08-30 21:02:44.293602	  116.32   38  4.31  14  4 82
2020-08-30 21:02:45.295612	   58.67   21  1.20   7  4 88
2020-08-30 21:02:46.297115	   61.49   83  4.98   6  4 90
2020-08-30 21:02:47.297344	   76.00    2  0.15   5  4 91
2020-08-30 21:02:48.297728	   56.90  301 16.72   8  4 88
2020-08-30 21:02:49.302826	   20.73   77  1.55   7  4 89
2020-08-30 21:02:50.302947	    5.49  105  0.56   7  4 89
2020-08-30 21:02:51.303215	  243.43   21  4.99   5  4 92
2020-08-30 21:02:52.303691	              disk0       cpu
2020-08-30 21:02:52.303732	    KB/t  tps  MB/s  us sy id
2020-08-30 21:02:52.303760	  912.00    9  8.01   5  4 91
2020-08-30 21:02:53.303808	   27.79  356  9.66   5  4 91
2020-08-30 21:02:54.305102	    0.00    0  0.00   4  4 92
2020-08-30 21:02:55.305241	   32.00    1  0.03   5  4 91
2020-08-30 21:02:56.305374	   28.00    1  0.03   6  4 91
2020-08-30 21:02:57.305456	    0.00    0  0.00  10  4 86
2020-08-30 21:02:58.308508	  212.10  123 25.40  14  4 82
2020-08-30 21:02:59.308681	   63.00   20  1.23   5  4 91
2020-08-30 21:03:00.311921	   90.64   44  3.88  12  4 84
2020-08-30 21:03:01.312268	   57.64   22  1.24   9  4 87
2020-08-30 21:03:02.312393	  192.00    3  0.56   6  4 90
2020-08-30 21:03:03.312868	    6.00   10  0.06   5  4 91
2020-08-30 21:03:04.316471	    0.00    0  0.00   5  4 92
2020-08-30 21:03:05.321060	  116.28   29  3.28  14  4 82
2020-08-30 21:03:06.321360	   57.64   22  1.24   7  4 89
2020-08-30 21:03:07.322305	    0.00    0  0.00   5  4 91
2020-08-30 21:03:08.322617	    0.00    0  0.00   5  4 91
2020-08-30 21:03:09.322747	  115.11   27  3.03  13  4 82
2020-08-30 21:03:10.325749	   55.48   23  1.24   6  4 90
2020-08-30 21:03:11.325844	   67.57  131  8.64   5  4 91
2020-08-30 21:03:12.326620	              disk0       cpu
2020-08-30 21:03:12.326708	    KB/t  tps  MB/s  us sy id
2020-08-30 21:03:12.326737	    0.00    0  0.00   5  4 91
2020-08-30 21:03:13.327893	   63.87   63  3.92   5  4 91
2020-08-30 21:03:14.328058	    6.67    3  0.02   5  5 90
2020-08-30 21:03:15.328256	   70.43   51  3.51   8  4 88
2020-08-30 21:03:16.328733	   12.00    1  0.01   4  4 92
2020-08-30 21:03:17.329525	    0.00    0  0.00   4  4 92
2020-08-30 21:03:18.329668	   49.33   18  0.87   4  4 92
2020-08-30 21:03:19.329825	    0.00    0  0.00   4  4 92
2020-08-30 21:03:20.329917	   33.86   56  1.85   5  5 90
2020-08-30 21:03:21.335066	    4.00    1  0.00   4  4 92
2020-08-30 21:03:22.335898	    0.00    0  0.00   4  4 92
2020-08-30 21:03:23.336120	   37.58  274 10.05   4  4 92
2020-08-30 21:03:24.336690	   36.25  224  7.93   6  5 90
2020-08-30 21:03:25.340858	    9.00    4  0.04   4  4 92
2020-08-30 21:03:26.344315	  141.08  111 15.24   5  4 91
2020-08-30 21:03:27.344434	   70.40    5  0.34   4  4 92
2020-08-30 21:03:28.346048	    4.00    1  0.00   4  4 92
2020-08-30 21:03:29.346155	    0.00    0  0.00   4  4 92
2020-08-30 21:03:30.346376	    0.00    0  0.00   4  4 92
2020-08-30 21:03:31.348957	   88.00    3  0.26   4  4 92
