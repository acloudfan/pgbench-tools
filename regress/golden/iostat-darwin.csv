collected,value,metric
2020-08-30 21:02:32.283212,26.49,disk0_KB/t
2020-08-30 21:02:32.283212,18.0,disk0_tps
2020-08-30 21:02:32.283212,0.47,disk0_MB/s
2020-08-30 21:02:32.283212,2.0,us
2020-08-30 21:02:32.283212,1.0,sy
2020-08-30 21:02:32.283212,96.0,id
2020-08-30 21:02:33.283271,28.4,disk0_KB/t
2020-08-30 21:02:33.283271,20.0,disk0_tps
2020-08-30 21:02:33.283271,0.55,disk0_MB/s
2020-08-30 21:02:33.283271,12.0,us
2020-08-30 21:02:33.283271,4.0,sy
2020-08-30 21:02:33.283271,84.0,id
2020-08-30 21:02:34.286027,13.6,disk0_KB/t
2020-08-30 21:02:34.286027,10.0,disk0_tps
2020-08-30 21:02:34.286027,0.13,disk0_MB/s
2020-08-30 21:02:34.286027,11.0,us
2020-08-30 21:02:34.286027,4.0,sy
2020-08-30 21:02:34.286027,85.0,id
2020-08-30 21:02:35.286896,30.96,disk0_KB/t
2020-08-30 21:02:35.286896,23.0,disk0_tps
2020-08-30 21:02:35.286896,0.69,disk0_MB/s
2020-08-30 21:02:35.286896,9.0,us
2020-08-30 21:02:35.286896,4.0,sy
2020-08-30 21:02:35.286896,87.0,id
2020-08-30 21:02:36.290639,90.0,disk0_KB/t
2020-08-30 21:02:36.290639,4.0,disk0_tps
2020-08-30 21:02:36.290639,0.35,disk0_MB/s
2020-08-30 21:02:36.290639,6.0,us
2020-08-30 21:02:36.290639,4.0,sy
2020-08-30 21:02:36.290639,90.0,id
2020-08-30 21:02:37.290742,23.37,disk0_KB/t
2020-08-30 21:02:37.290742,63.0,disk0_tps
2020-08-30 21:02:37.290742,1.44,disk0_MB/s
2020-08-30 21:02:37.290742,7.0,us
2020-08-30 21:02:37.290742,4.0,sy
2020-08-30 21:02:37.290742,89.0,id
2020-08-30 21:02:38.291489,166.61,disk0_KB/t
2020-08-30 21:02:38.291489,46.0,disk0_tps
2020-08-30 21:02:38.291489,7.48,disk0_MB/s
2020-08-30 21:02:38.291489,7.0,us
2020-08-30 21:02:38.291489,4.0,sy
2020-08-30 21:02:38.291489,89.0,id
2020-08-30 21:02:39.291605,0.0,disk0_KB/t
2020-08-30 21:02:39.291605,0.0,disk0_tps
2020-08-30 21:02:39.291605,0.0,disk0_MB/s
2020-08-30 21:02:39.291605,7.0,us
2020-08-30 21:02:39.291605,4.0,sy
2020-08-30 21:02:39.291605,89.0,id
2020-08-30 21:02:40.291813,24.0,disk0_KB/t
2020-08-30 21:02:40.291813,3.0,disk0_tps
2020-08-30 21:02:40.291813,0.07,disk0_MB/s
2020-08-30 21:02:40.291813,5.0,us
2020-08-30 21:02:40.291813,4.0,sy
2020-08-30 21:02:40.291813,91.0,id
2020-08-30 21:02:41.291947,4.83,disk0_KB/t
2020-08-30 21:02:41.291947,247.0,disk0_tps
2020-08-30 21:02:41.291947,1.16,disk0_MB/s
2020-08-30 21:02:41.291947,6.0,us
2020-08-30 21:02:41.291947,4.0,sy
2020-08-30 21:02:41.291947,90.0,id
2020-08-30 21:02:42.292114,62.67,disk0_KB/t
2020-08-30 21:02:42.292114,12.0,disk0_tps
2020-08-30 21:02:42.292114,0.73,disk0_MB/s
2020-08-30 21:02:42.292114,7.0,us
2020-08-30 21:02:42.292114,4.0,sy
2020-08-30 21:02:42.292114,88.0,id
2020-08-30 21:02:43.292373,0.0,disk0_KB/t
2020-08-30 21:02:43.292373,0.0,disk0_tps
2020-08-30 21:02:43.292373,0.0,disk0_MB/s
2020-08-30 21:02:43.292373,5.0,us
2020-08-30 21:02:43.292373,4.0,sy
2020-08-30 21:02:43.292373,91.0,id
2020-08-30 21:02:44.293602,116.32,disk0_KB/t
2020-08-30 21:02:44.293602,38.0,disk0_tps
2020-08-30 21:02:44.293602,4.31,disk0_MB/s
2020-08-30 21:02:44.293602,14.0,us
2020-08-30 21:02:44.293602,4.0,sy
2020-08-30 21:02:44.293602,82.0,id
2020-08-30 21:02:45.295612,58.67,disk0_KB/t
2020-08-30 21:02:45.295612,21.0,disk0_tps
2020-08-30 21:02:45.295612,1.2,disk0_MB/s
2020-08-30 21:02:45.295612,7.0,us
2020-08-30 21:02:45.295612,4.0,sy
2020-08-30 21:02:45.295612,88.0,id
2020-08-30 21:02:46.297115,61.49,disk0_KB/t
2020-08-30 21:02:46.297115,83.0,disk0_tps
2020-08-30 21:02:46.297115,4.98,disk0_MB/s
2020-08-30 21:02:46.297115,6.0,us
2020-08-30 21:02:46.297115,4.0,sy
2020-08-30 21:02:46.297115,90.0,id
2020-08-30 21:02:47.297344,76.0,disk0_KB/t
2020-08-30 21:02:47.297344,2.0,disk0_tps
2020-08-30 21:02:47.297344,0.15,disk0_MB/s
2020-08-30 21:02:47.297344,5.0,us
2020-08-30 21:02:47.297344,4.0,sy
2020-08-30 21:02:47.297344,91.0,id
2020-08-30 21:02:48.297728,56.9,disk0_KB/t
2020-08-30 21:02:48.297728,301.0,disk0_tps
2020-08-30 21:02:48.297728,16.72,disk0_MB/s
2020-08-30 21:02:48.297728,8.0,us
2020-08-30 21:02:48.297728,4.0,sy
2020-08-30 21:02:48.297728,88.0,id
2020-08-30 21:02:49.302826,20.73,disk0_KB/t
2020-08-30 21:02:49.302826,77.0,disk0_tps
2020-08-30 21:02:49.302826,1.55,disk0_MB/s
2020-08-30 21:02:49.302826,7.0,us
2020-08-30 21:02:49.302826,4.0,sy
2020-08-30 21:02:49.302826,89.0,id
2020-08-30 21:02:50.302947,5.49,disk0_KB/t
2020-08-30 21:02:50.302947,105.0,disk0_tps
2020-08-30 21:02:50.302947,0.56,disk0_MB/s
2020-08-30 21:02:50.302947,7.0,us
2020-08-30 21:02:50.302947,4.0,sy
2020-08-30 21:02:50.302947,89.0,id
2020-08-30 21:02:51.303215,243.43,disk0_KB/t
2020-08-30 21:02:51.303215,21.0,disk0_tps
2020-08-30 21:02:51.303215,4.99,disk0_MB/s
2020-08-30 21:02:51.303215,5.0,us
2020-08-30 21:02:51.303215,4.0,sy
2020-08-30 21:02:51.303215,92.0,id
2020-08-30 21:02:52.303760,912.0,disk0_KB/t
2020-08-30 21:02:52.303760,9.0,disk0_tps
2020-08-30 21:02:52.303760,8.01,disk0_MB/s
2020-08-30 21:02:52.303760,5.0,us
2020-08-30 21:02:52.303760,4.0,sy
2020-08-30 21:02:52.303760,91.0,id
2020-08-30 21:02:53.303808,27.79,disk0_KB/t
2020-08-30 21:02:53.303808,356.0,disk0_tps
2020-08-30 21:02:53.303808,9.66,disk0_MB/s
2020-08-30 21:02:53.303808,5.0,us
2020-08-30 21:02:53.303808,4.0,sy
2020-08-30 21:02:53.303808,91.0,id
2020-08-30 21:02:54.305102,0.0,disk0_KB/t
2020-08-30 21:02:54.305102,0.0,disk0_tps
2020-08-30 21:02:54.305102,0.0,disk0_MB/s
2020-08-30 21:02:54.305102,4.0,us
2020-08-30 21:02:54.305102,4.0,sy
2020-08-30 21:02:54.305102,92.0,id
2020-08-30 21:02:55.305241,32.0,disk0_KB/t
2020-08-30 21:02:55.305241,1.0,disk0_tps
2020-08-30 21:02:55.305241,0.03,disk0_MB/s
2020-08-30 21:02:55.305241,5.0,us
2020-08-30 21:02:55.305241,4.0,sy
2020-08-30 21:02:55.305241,91.0,id
2020-08-30 21:02:56.305374,28.0,disk0_KB/t
2020-08-30 21:02:56.305374,1.0,disk0_tps
2020-08-30 21:02:56.305374,0.03,disk0_MB/s
2020-08-30 21:02:56.305374,6.0,us
2020-08-30 21:02:56.305374,4.0,sy
2020-08-30 21:02:56.305374,91.0,id
2020-08-30 21:02:57.305456,0.0,disk0_KB/t
2020-08-30 21:02:57.305456,0.0,disk0_tps
2020-08-30 21:02:57.305456,0.0,disk0_MB/s
2020-08-30 21:02:57.305456,10.0,us
2020-08-30 21:02:57.305456,4.0,sy
2020-08-30 21:02:57.305456,86.0,id
2020-08-30 21:02:58.308508,212.1,disk0_KB/t
2020-08-30 21:02:58.308508,123.0,disk0_tps
2020-08-30 21:02:58.308508,25.4,disk0_MB/s
2020-08-30 21:02:58.308508,14.0,us
2020-08-30 21:02:58.308508,4.0,sy
2020-08-30 21:02:58.308508,82.0,id
2020-08-30 21:02:59.308681,63.0,disk0_KB/t
2020-08-30 21:02:59.308681,20.0,disk0_tps
2020-08-30 21:02:59.308681,1.23,disk0_MB/s
2020-08-30 21:02:59.308681,5.0,us
2020-08-30 21:02:59.308681,4.0,sy
2020-08-30 21:02:59.308681,91.0,id
2020-08-30 21:03:00.311921,90.64,disk0_KB/t
2020-08-30 21:03:00.311921,44.0,disk0_tps
2020-08-30 21:03:00.311921,3.88,disk0_MB/s
2020-08-30 21:03:00.311921,12.0,us
2020-08-30 21:03:00.311921,4.0,sy
2020-08-30 21:03:00.311921,84.0,id
2020-08-30 21:03:01.312268,57.64,disk0_KB/t
2020-08-30 21:03:01.312268,22.0,disk0_tps
2020-08-30 21:03:01.312268,1.24,disk0_MB/s
2020-08-30 21:03:01.312268,9.0,us
2020-08-30 21:03:01.312268,4.0,sy
2020-08-30 21:03:01.312268,87.0,id
2020-08-30 21:03:02.312393,192.0,disk0_KB/t
2020-08-30 21:03:02.312393,3.0,disk0_tps
2020-08-30 21:03:02.312393,0.56,disk0_MB/s
2020-08-30 21:03:02.312393,6.0,us
2020-08-30 21:03:02.312393,4.0,sy
2020-08-30 21:03:02.312393,90.0,id
2020-08-30 21:03:03.312868,6.0,disk0_KB/t
2020-08-30 21:03:03.312868,10.0,disk0_tps
2020-08-30 21:03:03.312868,0.06,disk0_MB/s
2020-08-30 21:03:03.312868,5.0,us
2020-08-30 21:03:03.312868,4.0,sy
2020-08-30 21:03:03.312868,91.0,id
2020-08-30 21:03:04.316471,0.0,disk0_KB/t
2020-08-30 21:03:04.316471,0.0,disk0_tps
2020-08-30 21:03:04.316471,0.0,disk0_MB/s
2020-08-30 21:03:04.316471,5.0,us
2020-08-30 21:03:04.316471,4.0,sy
2020-08-30 21:03:04.316471,92.0,id
2020-08-30 21:03:05.321060,116.28,disk0_KB/t
2020-08-30 21:03:05.321060,29.0,disk0_tps
2020-08-30 21:03:05.321060,3.28,disk0_MB/s
2020-08-30 21:03:05.321060,14.0,us
2020-08-30 21:03:05.321060,4.0,sy
2020-08-30 21:03:05.321060,82.0,id
2020-08-30 21:03:06.321360,57.64,disk0_KB/t
2020-08-30 21:03:06.321360,22.0,disk0_tps
2020-08-30 21:03:06.321360,1.24,disk0_MB/s
2020-08-30 21:03:06.321360,7.0,us
2020-08-30 21:03:06.321360,4.0,sy
2020-08-30 21:03:06.321360,89.0,id
2020-08-30 21:03:07.322305,0.0,disk0_KB/t
2020-08-30 21:03:07.322305,0.0,disk0_tps
2020-08-30 21:03:07.322305,0.0,disk0_MB/s
2020-08-30 21:03:07.322305,5.0,us
2020-08-30 21:03:07.322305,4.0,sy
2020-08-30 21:03:07.322305,91.0,id
2020-08-30 21:03:08.322617,0.0,disk0_KB/t
2020-08-30 21:03:08.322617,0.0,disk0_tps
2020-08-30 21:03:08.322617,0.0,disk0_MB/s
2020-08-30 21:03:08.322617,5.0,us
2020-08-30 21:03:08.322617,4.0,sy
2020-08-30 21:03:08.322617,91.0,id
2020-08-30 21:03:09.322747,115.11,disk0_KB/t
2020-08-30 21:03:09.322747,27.0,disk0_tps
2020-08-30 21:03:09.322747,3.03,disk0_MB/s
2020-08-30 21:03:09.322747,13.0,us
2020-08-30 21:03:09.322747,4.0,sy
2020-08-30 21:03:09.322747,82.0,id
2020-08-30 21:03:10.325749,55.48,disk0_KB/t
2020-08-30 21:03:10.325749,23.0,disk0_tps
2020-08-30 21:03:10.325749,1.24,disk0_MB/s
2020-08-30 21:03:10.325749,6.0,us
2020-08-30 21:03:10.325749,4.0,sy
2020-08-30 21:03:10.325749,90.0,id
2020-08-30 21:03:11.325844,67.57,disk0_KB/t
2020-08-30 21:03:11.325844,131.0,disk0_tps
2020-08-30 21:03:11.325844,8.64,disk0_MB/s
2020-08-30 21:03:11.325844,5.0,us
2020-08-30 21:03:11.325844,4.0,sy
2020-08-30 21:03:11.325844,91.0,id
2020-08-30 21:03:12.326737,0.0,disk0_KB/t
2020-08-30 21:03:12.326737,0.0,disk0_tps
2020-08-30 21:03:12.326737,0.0,disk0_MB/s
2020-08-30 21:03:12.326737,5.0,us
2020-08-30 21:03:12.326737,4.0,sy
2020-08-30 21:03:12.326737,91.0,id
2020-08-30 21:03:13.327893,63.87,disk0_KB/t
2020-08-30 21:03:13.327893,63.0,disk0_tps
2020-08-30 21:03:13.327893,3.92,disk0_MB/s
2020-08-30 21:03:13.327893,5.0,us
2020-08-30 21:03:13.327893,4.0,sy
2020-08-30 21:03:13.327893,91.0,id
2020-08-30 21:03:14.328058,6.67,disk0_KB/t
2020-08-30 21:03:14.328058,3.0,disk0_tps
2020-08-30 21:03:14.328058,0.02,disk0_MB/s
2020-08-30 21:03:14.328058,5.0,us
2020-08-30 21:03:14.328058,5.0,sy
2020-08-30 21:03:14.328058,90.0,id
2020-08-30 21:03:15.328256,70.43,disk0_KB/t
2020-08-30 21:03:15.328256,51.0,disk0_tps
2020-08-30 21:03:15.328256,3.51,disk0_MB/s
2020-08-30 21:03:15.328256,8.0,us
2020-08-30 21:03:15.328256,4.0,sy
2020-08-30 21:03:15.328256,88.0,id
2020-08-30 21:03:16.328733,12.0,disk0_KB/t
2020-08-30 21:03:16.328733,1.0,disk0_tps
2020-08-30 21:03:16.328733,0.01,disk0_MB/s
2020-08-30 21:03:16.328733,4.0,us
2020-08-30 21:03:16.328733,4.0,sy
2020-08-30 21:03:16.328733,92.0,id
2020-08-30 21:03:17.329525,0.0,disk0_KB/t
2020-08-30 21:03:17.329525,0.0,disk0_tps
2020-08-30 21:03:17.329525,0.0,disk0_MB/s
2020-08-30 21:03:17.329525,4.0,us
2020-08-30 21:03:17.329525,4.0,sy
2020-08-30 21:03:17.329525,92.0,id
2020-08-30 21:03:18.329668,49.33,disk0_KB/t
2020-08-30 21:03:18.329668,18.0,disk0_tps
2020-08-30 21:03:18.329668,0.87,disk0_MB/s
2020-08-30 21:03:18.329668,4.0,us
2020-08-30 21:03:18.329668,4.0,sy
2020-08-30 21:03:18.329668,92.0,id
2020-08-30 21:03:19.329825,0.0,disk0_KB/t
2020-08-30 21:03:19.329825,0.0,disk0_tps
2020-08-30 21:03:19.329825,0.0,disk0_MB/s
2020-08-30 21:03:19.329825,4.0,us
2020-08-30 21:03:19.329825,4.0,sy
2020-08-30 21:03:19.329825,92.0,id
2020-08-30 21:03:20.329917,33.86,disk0_KB/t
2020-08-30 21:03:20.329917,56.0,disk0_tps
2020-08-30 21:03:20.329917,1.85,disk0_MB/s
2020-08-30 21:03:20.329917,5.0,us
2020-08-30 21:03:20.329917,5.0,sy
2020-08-30 21:03:20.329917,90.0,id
2020-08-30 21:03:21.335066,4.0,disk0_KB/t
2020-08-30 21:03:21.335066,1.0,disk0_tps
2020-08-30 21:03:21.335066,0.0,disk0_MB/s
2020-08-30 21:03:21.335066,4.0,us
2020-08-30 21:03:21.335066,4.0,sy
2020-08-30 21:03:21.335066,92.0,id
2020-08-30 21:03:22.335898,0.0,disk0_KB/t
2020-08-30 21:03:22.335898,0.0,disk0_tps
2020-08-30 21:03:22.335898,0.0,disk0_MB/s
2020-08-30 21:03:22.335898,4.0,us
2020-08-30 21:03:22.335898,4.0,sy
2020-08-30 21:03:22.335898,92.0,id
2020-08-30 21:03:23.336120,37.58,disk0_KB/t
2020-08-30 21:03:23.336120,274.0,disk0_tps
2020-08-30 21:03:23.336120,10.05,disk0_MB/s
2020-08-30 21:03:23.336120,4.0,us
2020-08-30 21:03:23.336120,4.0,sy
2020-08-30 21:03:23.336120,92.0,id
2020-08-30 21:03:24.336690,36.25,disk0_KB/t
2020-08-30 21:03:24.336690,224.0,disk0_tps
2020-08-30 21:03:24.336690,7.93,disk0_MB/s
2020-08-30 21:03:24.336690,6.0,us
2020-08-30 21:03:24.336690,5.0,sy
2020-08-30 21:03:24.336690,90.0,id
2020-08-30 21:03:25.340858,9.0,disk0_KB/t
2020-08-30 21:03:25.340858,4.0,disk0_tps
2020-08-30 21:03:25.340858,0.04,disk0_MB/s
2020-08-30 21:03:25.340858,4.0,us
2020-08-30 21:03:25.340858,4.0,sy
2020-08-30 21:03:25.340858,92.0,id
2020-08-30 21:03:26.344315,141.08,disk0_KB/t
2020-08-30 21:03:26.344315,111.0,disk0_tps
2020-08-30 21:03:26.344315,15.24,disk0_MB/s
2020-08-30 21:03:26.344315,5.0,us
2020-08-30 21:03:26.344315,4.0,sy
2020-08-30 21:03:26.344315,91.0,id
2020-08-30 21:03:27.344434,70.4,disk0_KB/t
2020-08-30 21:03:27.344434,5.0,disk0_tps
2020-08-30 21:03:27.344434,0.34,disk0_MB/s
2020-08-30 21:03:27.344434,4.0,us
2020-08-30 21:03:27.344434,4.0,sy
2020-08-30 21:03:27.344434,92.0,id
2020-08-30 21:03:28.346048,4.0,disk0_KB/t
2020-08-30 21:03:28.346048,1.0,disk0_tps
2020-08-30 21:03:28.346048,0.0,disk0_MB/s
2020-08-30 21:03:28.346048,4.0,us
2020-08-30 21:03:28.346048,4.0,sy
2020-08-30 21:03:28.346048,92.0,id
2020-08-30 21:03:29.346155,0.0,disk0_KB/t
2020-08-30 21:03:29.346155,0.0,disk0_tps
2020-08-30 21:03:29.346155,0.0,disk0_MB/s
2020-08-30 21:03:29.346155,4.0,us
2020-08-30 21:03:29.346155,4.0,sy
2020-08-30 21:03:29.346155,92.0,id
2020-08-30 21:03:30.346376,0.0,disk0_KB/t
2020-08-30 21:03:30.346376,0.0,disk0_tps
2020-08-30 21:03:30.346376,0.0,disk0_MB/s
2020-08-30 21:03:30.346376,4.0,us
2020-08-30 21:03:30.346376,4.0,sy
2020-08-30 21:03:30.346376,92.0,id
2020-08-30 21:03:31.348957,88.0,disk0_KB/t
2020-08-30 21:03:31.348957,3.0,disk0_tps
2020-08-30 21:03:31.348957,0.26,disk0_MB/s
2020-08-30 21:03:31.348957,4.0,us
2020-08-30 21:03:31.348957,4.0,sy
2020-08-30 21:03:31.348957,92.0,id
